calculus/integration-substitution.md

```diff
@@ -1 +1,45 @@
 # Integration using substitution
+## Skill Builder
+### Find General Solutions for:
+(a) $y' = (x+2)ln(x)$
+
+  $y = \int (x+2)\ln(x) \ dx$  
+    
+  $u = ln(x)$ $dv = (x+2)dx$  
+  $du = \frac{1}{x}dx$  $v = \frac{x^2}{2} + 2x$
+    
+ $=(\frac{x^2}{2} + 2x)ln(x)- \int (\frac{x^2}{2} + 2x) \frac{1}{x}\ dx$    
+ <br> $\to (\frac{x^2}{2} + 2x)ln(x)-\frac{x^2}{4}-2x+C$  
+<br>
+<br>
+
+(b) $y' = x^5sin(x)$<br>
```

Comment:
```suggestion
(b) $y' = x^5\sin(x)$
```
Like with $\ln(x)$, the sin and cos functions need to be written as `\sin` and `\cos`.